analyzer-core/src/main/java/ffb/analyzer/core/queue/PrioritizedTask.java

```diff
@@ -3,25 +3,43 @@
 import java.util.function.Consumer;
 
 public class PrioritizedTask<T> extends Task<T> implements Comparable<PrioritizedTask<T>> {
-    private final Integer priority;
+    public enum Priority {
```

Comment:
`Lowest` and `Highest`?

Reply:
This sort of ties back to what I mentioned on Friday about leaving room to expand. If we name them Lowest and Highest we're restricting ourselves to never have room to increase/decrease at the boundaries (unless we want Lowestest or EvenLowester - or possibly looooooooowest). I think Lower/Higher make their positions clear, while still giving us the flexibility to expand to Lowest/Highest in the future if need be with minimal impact.